### PR TITLE
Update OLM CSV description.

### PR DESCRIPTION
**Pull request by chancez**
Supersedes #586

### CODE REVIEW

**Comment by chancez** [comment]
origin installer based tests are flaking due to quota issue it seems, so
/override ci/prow/integration-aws-metering

File: manifests/deploy/common-olm-values.yaml [hunk 1]
```diff
@@ -2,9 +2,35 @@ csv:
...
+
+    Metering runs a big data stack to crunch your data and requires at least 4GB of RAM and 2 CPU cores. At least one Node should have 2GB of RAM and 1 CPU core. Memory and CPU consumption may often be lower, but will spike when running reports, or collecting data for larger clusters.
+
+    ### Common Configurations
+
+    Metering works out of the box without any customizations or configuration. Read the [full documentation](https://github.com/operator-framework/operator-metering/blob/master/Documentation/metering-config.md) for more details.
+
+    * **Use a specific StorageClass** - Follow the example to [use your own StorageClass](https://github.com/operator-framework/operator-metering/blob/master/manifests/metering-config/custom-storage.yaml) instead of the Dynamic Provisioner.
+
+    * **Store data in S3 instead of PV** - Store your report output [in an S3 bucket](https://github.com/operator-framework/operator-metering/blob/master/Documentation/configuring-storage.md#storing-data-in-s3) instead of a PresistentVolume.
```
Comment (tschuy) [inline manifests/deploy/common-olm-values.yaml]: PersistentVolume

File: manifests/custom-resource-definitions/reportdatasource.crd.yaml [hunk 2]
```diff
@@ -3,8 +3,9 @@ kind: CustomResourceDefinition
 metadata:
   name: reportdatasources.metering.openshift.io
   annotations:
-    catalog.app.coreos.com/displayName: "Metering data source"
-    catalog.app.coreos.com/description: "A resource describing a source of data for usage by Report Generation Queries"
+    catalog.app.coreos.com/displayName: Metering Data Source
+    catalog.app.coreos.com/description: A resource describing a source of data for usage by Report Generation Queries.
+    catalog.app.coreos.com/weight: 4
 spec:
```
Comment (EmilyM1) [inline manifests/custom-resource-definitions/reportdatasource.crd.yaml]: Do these need to be strings?
catalog.app.coreos.com/displayName
catalog.app.coreos.com/description

File: manifests/deploy/common-olm-values.yaml [hunk 3]
```diff
@@ -2,9 +2,35 @@ csv:
   name : metering-operator.vlatest
   displayName: Metering
   description: |
-    **Metering** can generate reports based on historical usage data from a cluster, providing accountability for how resources have been used.
+    Operator Metering is a chargeback and reporting tool to provide accountability for how resources are used across a cluster. Cluster admins can schedule reports based on historical usage data by Pod, Namespace, and cluster wide. Operator Metering is part of the [Operator Framework](https://coreos.com/blog/introducing-operator-framework-metering).
```
Comment (EmilyM1) [inline manifests/deploy/common-olm-values.yaml]: by Pod, Namespace, and Cluster wide

File: manifests/custom-resource-definitions/reportdatasource.crd.yaml (hunk 2 above)
Comment (chancez) [reply manifests/custom-resource-definitions/reportdatasource.crd.yaml]: Those are strings, yaml doesn't require quotes. That being said; the `weight` probably does need quotes.

**Comment by chancez** [comment]
e2e has flaked when trying to patch machinesets multiple times, im bypassing it since integration passed.
/override ci/prow/e2e-aws-metering

**Pull request merged**